packages/host/app/services/realm.ts

```diff
@@ -641,7 +638,7 @@ export default class RealmService extends Service {
   private identifyRealm = task(
     { maxConcurrency: 1, enqueue: true },
     async (url: string): Promise<void> => {
-      if (this.knownRealm(url)) {
+      if (this.knownRealm(url, false)) {
```

Comment:
Wondering why this was needed

Reply:
Updated.

Reply:
We need this to avoid the error: "Attempted to update a tracked variable, but it had already been used previously in the same computation." Also, we don't need to use the tracked variable in this task.